src/applications/personalization/profile/components/ProfileWrapper.jsx

```diff
@@ -93,11 +107,21 @@ const ProfileWrapper = ({
             />
             <div className="vads-l-row">
               <div className="vads-u-display--none medium-screen:vads-u-display--block vads-l-col--3 vads-u-padding-left--2">
-                <ProfileSubNav
-                  routes={routesForNav}
-                  isLOA3={isLOA3}
-                  isInMVI={isInMVI}
-                />
+                <nav className="va-subnav" aria-labelledby="subnav-header">
+                  <div>
+                    <h2
+                      id="subnav-header"
+                      className="vads-u-font-size--h4 vads-u-margin-top--0 vads-u-margin-bottom--0 vads-u-padding-y--2"
+                    >
+                      Profile <span className="sr-only">menu</span>
+                    </h2>
```

Comment:
I added this here to minimize changes. The ProfileSubNav component, once Paperless Delivery is live, will not require a separate mobile and non-mobile nav components as the VaSidenav shows a mobile design at 767px.